packages/_private/e2e-demo-test/package.json

```
@@ -0,0 +1,42 @@
+{
+  "name": "@synergy-design-system/vanilla-demo",
```

Copy and paste, should be:

```suggestion
  "name": "@synergy-design-system/e2e-demo-test",
```